hudi-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -43,72 +47,110 @@
 
   private static final Logger LOG = LogManager.getLogger(DFSPropertiesConfiguration.class);
 
+  private static final String DEFAULT_PROPERTIES_FILE = "hudi-defaults.conf";
```

Comment:
can this be public? since it's always the same file name to be read? this info should be made public then